src/__internal__/label/label.component.tsx

```diff
@@ -62,7 +62,7 @@ const tooltipPosition = ({
 };
 
 export const Label = ({
-  align = "right",
+  align = "left",
```

Comment:
comment: this is causing some of the baselines to update in [chromatic](https://www.chromatic.com/test?appId=5ecf782fe724630022d27d7d&id=67448128300e429aa2d30e59) so I think we need to be a little more nuanced here. We may need to calculate the value of the prop inside the component as opposed to in the prop destructuring

```
// label.component
export const Label = ({
  align,
  ...
}) => {
  const alignment = inline && !align ? "right" : align;
 ...
 <StyledLabelContainer
      data-role="label-container"
      align={alignment}
      inline={inline}
      width={width}
      optional={optional}
      pr={pr}
      pl={pl}
      className={className}
    >

```

```
// label.style
${({ align }) => css`
   justify-content: ${align !== "right" ? "flex-start" : "flex-end"};
`}

Reply:
Some of the baselines are still going to flag changes, for example I notice in numeral-date we are setting labelAlign="right", we should delete them for the `Required` and `isOptional` stories as it doesn't make sense that they're being set

Reply:
Ok, sounds good!

Reply:
Yeah, I agree here, this should reduce the changeset + the changes you've made to our pw tests